third_party/move/move-compiler-v2/tests/checking/anonymous_fields/named_tuple_construct_invalid.move

```diff
@@ -0,0 +1,15 @@
+module 0x42::test {
+    struct S(u8);
+
+    fun arity_mismatch0(): S {
+        S()
```

Comment:
Add a similar test but with S having 2 fields and you providing 1.  Also for an assignment.  1/0 might be a special case, test enough for confidence that both corner cases and typical cases will work.  A single test case isn't enough to see a rule.

Reply:
done